js/scripts.js

```diff
@@ -0,0 +1,85 @@
+
+
+var state = {
+	"one": false,
+	"two": false,
+	"three": false,
+	"four": false,
+	"five": false,
+	"six": false,
+	"seven": false,
+	"eight": false,
+	"nine": false,
+};
+document.addEventListener("DOMContentLoaded", function(){
+	document.getElementById("one").addEventListener("click", buttonListener1);
```

Comment:
You can put these in a function, similar to how you did with the clear clrListener:
```
var setBoard = function(){
	for (num in state) {
		document.getElementById(num).addEventListener("click", buttonListener1);
	}
}
document.addEventListener("DOMContentLoaded", function(){
	setBoard()
});
```